environment/elisp/format-f90.el

```diff
@@ -0,0 +1,43 @@
+;;; File  : format-f90.el
+;;; Author: Kelly Thompson, kgt@lanl.gov
+;;; Date  : Wednesday, Sep 05, 2018, 16:01 pm
+
+;;; Description:
+
+;;; With format-f90.sh, laod a file in Emacs, apply standard Draco
```

Comment:
`laod` -> `load`

Reply:
I'll fix this in my next commit (already fixed locally).  Since it doesn't break anything, I'll let it be for this PR.